src/main/resources/allowed-labels.properties

```diff
@@ -22,6 +23,7 @@ cloud
 cluster
 cmp
 codebuild
+confluence
```

Comment:
I found three plugins that seem like good candidates for the confluence label:

* [Confluence publisher](https://plugins.jenkins.io/confluence-publisher/)
* [Release helper](https://plugins.jenkins.io/release-helper/)
* [Docktor](https://plugins.jenkins.io/doktor/) 

There is also a reference to a commercial product from "Cloudpipes" but no reference to Cloudpipes on any Jenkins site that I can see.

Are there other Confluence related plugins that I have missed?

Reply:
https://github.com/jenkinsci/confluence-pipeline-steps-plugin but it is archived